mindsdb/repository.yaml

```diff
@@ -0,0 +1,12 @@
+name: mindsdb
+description: MindsDB is a Server for Artificial Intelligence Logic, enabling developers to ship AI powered systems from prototyping & experimentation to production in a fast & scalable way.
+category: DATA
+private: false
+releaseStatus: BETA
+icon: plural/icons/mindsdb.png
+notes: plural/notes.tpl
+homepage: https://mindsdb.com
+gitUrl: https://github.com/mindsdb/mindsdb
+
+contributors:
+- Oghenemaro Akpobi
```

Comment:
this would need to be the email you use to login with plural actually

Reply:
> this would need to be the email you use to login with plural actually

okay